Sources/Splash/Output/AttributedStringOutputFormat.swift

```diff
@@ -94,21 +112,52 @@ private extension NSFont {
             .cfurlposixPathStyle,
             false
         )
-
+        
         guard let font = url.flatMap(CGDataProvider.init).flatMap(CGFont.init) else {
             return nil
         }
-
+        
         return CTFontCreateWithGraphicsFont(font, size, nil, nil)
     }
-
+    
     static func defaultFont(ofSize size: CGFloat) -> NSFont {
         guard let courier = loaded(from: "/Library/Fonts/Courier New.ttf", size: size) else {
             return .systemFont(ofSize: size)
         }
-
+        
         return courier
     }
 }
+#endif
+
+#if os(iOS)
+private extension NSMutableAttributedString {
+    func append(_ string: String, font: UIFont, color: Color) {
```

Comment:
If we typealias `UIFont/NSFont` and `UIColor/NSColor`, we should be able to reuse the same code for appending to `NSMutableAttributedString`. But it doesn't have to be part of this PR, I can fix that post-merge 🙂

Reply:
I can take care of that 👍 Any preferences how to call the typealias? Since there's already a `Font`, we could go for `SystemFont` or `PlatformFont` 🤔